stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -233,6 +234,10 @@ internal class ChatDomainImpl internal constructor(
 
         // load channel configs from Room into memory
         initJob = scope.async(scope.coroutineContext) {
+            clearState()
+            if (client.isSocketConnected()) {
+                setOnline()
+            }
```

Comment:
withContext could also be used here, this prevents having to make changes to our test codebase

Reply:
`initJob` is not going to be completed in test-side, blocking some of our tests that will be waiting `initJob` to be completed

Reply:
indeed